birdbox/microsite/forms.py

```diff
@@ -98,6 +98,17 @@ def __init__(self, *args, **kwargs):
 
 class BuildersChallengeForm(ContactFormBase):
     def __init__(self, *args, **kwargs):
+        interests = forms.MultipleChoiceField(
+            label=(""),
+            choices=(
+                (
+                    "mozilla-builders-application-2024",
```

Comment:
Hi @johnshaughnessy - is Basket/Braze set up to deal with this new interest name?

Reply:
Yes